docs/install/operator/configuring-serving-cr.md

```diff
@@ -411,3 +412,31 @@ spec:
       memory: 250Mi
       ephemeral-storage: 4Gi
 ```
+
+## Override system deployments
+
+If you would like to override some configurations for a specific deployment, you can override the configuration by using `spec.deployments` in CR.
```

Comment:
@nak3 which versions of the operator if this supported? 0.21, 0.20, 0.19 if it's supported then the change needs to be cherry pick to other branches after merging on master

Thank you

Reply:
Thank you! Yes, this is supported from 0.21 so we need to cherry pick to 0.21 branch.